src/app/search/search.component.ts

```diff
@@ -224,4 +224,9 @@ export class SearchComponent implements OnInit {
   loadMore() {
     this.doSearch(false);
   }
+
+  clearTerms() {
+    this.terms.clear();
+    this.router.navigate(['search']);
```

Comment:
Why do we need to re-navigate to the page?

Reply:
The search parameters, after you click search, get added to the url.
This clears it, so that if you refresh or click 'back' it doesnt repopulate the fields you just cleared.